radioapp/ui/ui_backlight.py

```diff
@@ -0,0 +1,62 @@
+from tkinter import *

```

Comment:
This file uses hard tabs, but we're following PEP8, which specifies 4 space tabs.